webdriver/tests/support/pdf.py

```diff
@@ -4,5 +4,5 @@
 def assert_pdf(value):
     data = decodebytes(value.encode())
 
-    assert data.startswith(b"%PDF-"), "Decoded data starts with the PDF signature"
-    assert data.endswith(b"%%EOF\n"), "Decoded data ends with the EOF flag"
+    assert data.startswith(b"%PDF-"), "Decoded data should start with the PDF signature"
+    assert data.endswith(b"%%EOF\n") or data.endswith(b"%%EOF"), "Decoded data should end with the EOF flag"
```

Comment:
Can you please add the Chrome bug as reference here please? Thanks.

Reply:
done